ngo/data/qweb_view.xml

```diff
@@ -0,0 +1,1818 @@
+                                                        info@yourcompany.example.com</a>
+                                                </span>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                    <div class="col-lg-3 pb24 o_colored_level">
+                                        <div class="s_social_media text-end o_not_editable"
+                                            data-snippet="s_social_media" data-name="Social Media">
+                                            <h5 class="s_social_media_title d-none">Follow Us</h5>
+                                            <a href="/website/social/facebook"
+                                                class="s_social_media_facebook" target="_blank"
+                                                aria-label="Facebook">
+                                                <i
+                                                    class="fa fa-1x fa-facebook rounded-circle shadow-sm o_editable_media"/>
+                                            </a>
+                                            <a href="/website/social/twitter"
+                                                class="s_social_media_twitter" target="_blank"
+                                                aria-label="X">
+                                                <i
+                                                    class="fa fa-1x fa-twitter rounded-circle shadow-sm o_editable_media"/>
+                                            </a>
+                                            <a href="/website/social/instagram"
+                                                class="s_social_media_instagram" target="_blank"
+                                                aria-label="Instagram">
+                                                <i
+                                                    class="fa fa-1x fa-instagram rounded-circle shadow-sm o_editable_media"/>
+                                            </a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                </xpath>
+            </data>
+        </field>
+        <field name="inherit_id" ref="website.layout"/>
+        <field name="key">website.template_footer_headline</field>
+        <field name="mode">extension</field>
+        <field name="name">Headline</field>
+        <field name="active" eval="True"/>
+        <field name="type">qweb</field>
+        <field name="website_id" ref="website.default_website"/>
+    </record>
+    <record id="website_homepage_qweb_view" model="ir.ui.view">
+        <field name="arch" type="xml">
+            <t name="Homepage" t-name="website.homepage">
```

Comment:
Some views are definitely demo data and should move to demo/
(of course not the reports and the one necessary for the business)